src/ontology/mondo-edit.obo

```diff
@@ -535490,10 +535514,13 @@ subset: clingen {source="MONDO:CLINGEN"}
 subset: gard_rare {source="GARD:10430", source="MONDO:GARD"}
 subset: nord_rare {source="NORD:1061", source="MONDO:NORD"}
 subset: rare
+synonym: "Dravet" EXACT [PMID:37498137]
 synonym: "Dravet syndrome" EXACT CLINGEN_LABEL [OMIM:607208]
 synonym: "DS" EXACT ABBREVIATION [Orphanet:33069]
 synonym: "myoclonic epilepsy, severe, of infancy" RELATED [GARD:0010430]
+synonym: "severe myoclonic epilepsy of infancy" EXACT [PMID:37498137]
```

Comment:
There are 2 synonyms that look the same, one is "related" and the other one is "exact". Could you please review and update accordingly? Thank you

Reply:
updated